rpcserver.go

```diff
@@ -6469,6 +6473,36 @@ func (r *rpcServer) ListMacaroonIDs(ctx context.Context,
 	return resp, nil
 }
 
+// DeleteMacaroonID removes a specific macaroon ID.
+func (r *rpcServer) DeleteMacaroonID(ctx context.Context,
+	req *lnrpc.DeleteMacaroonIDRequest) (*lnrpc.DeleteMacaroonIDResponse, error) {
+
+	rpcsLog.Debugf("[deletemacaroonid]")
+
+	// If the --no-macaroons flag is used to start lnd, the macaroon service
+	// is not initialized. Therefore we can't show any IDs.
+	if r.macService == nil {
+		return nil, fmt.Errorf("macaroon authentication disabled, " +
+			"remove --no-macaroons flag to enable")
+	}
+
+	// Convert root key id from uint64 to bytes. Because the DefaultRootKeyID is
+	// a digit 0 expressed in a byte slice of a string "0", we will keep the IDs
+	// in the same format - all must be digit, and must be a byte slice of
+	// string value of the digit.
+	rootKeyID := []byte(fmt.Sprint(req.RootKeyId))
```

Comment:
Should use `strconv.FormatUInt` here as well.

Reply:
my bad, fixed.